packages/chakra-components/src/ComboBox/useSelect.js

```diff
@@ -74,8 +74,11 @@ function useHoistedState(initialState, reducer) {
   return [state, setState];
 }
 
+let rapidTypeaheadDebouncer = null;
+
 export default function useSelect({
   multi,
+  isListBox,
   create,
   getCreateLabel = d => `Use "${d}"`,
   stateReducer = (old, newState, action) => newState,
```

Comment:
Can remove this create label.

Reply:
Please review only the MultiSelect folder, I believe we can delete this component and rename our MultiSelect folder to ComboBox once done.